.gitignore

```diff
@@ -30,3 +30,4 @@ config/bash.env
 .idea/sonarlint-state.xml
 phpstan/phpstan-baseline-temp.neon
 phpstan/phpstan-baseline-pr.neon
+docs/_site/
```

Comment:
If this is the output directory, and we ignore it. Then what is automatically committed?

Reply:
It is just there to ensure we do not accidentally push anything from the generated documents onto one of our normal branches. The directory itself is a separate git repository in the action itself.

One thing I may have forgotten is a `.gitignore` for the caches on the `gh-pages` branch, but we'll have to check that out after the action has run.